content/docs/user-guide/basic-concepts/dvc-project.md

```diff
@@ -12,7 +12,36 @@ match:
     repositories
   ]
 tooltip: >-
-  Initialized by running `dvc init` in the **workspace** (typically a Git
-  repository). It will contain the `.dvc/` directory, as well as `dvc.yaml` and
-  `.dvc` files created with commands such as `dvc add` or `dvc run`.
+  A DVC project is a directory initialized by `dvc init` that contains a `.dvc`
+  directory. It contains all <abbr>metafiles</abbr> related to project and also
+  includes the underlying <abbr>cache</abbr>.
+  [📖](/doc/user-guide/basic-concepts/dvc-project)
 ---
+
+# DVC Project
+
+A DVC project (or DVC repository) is a directory initialized by `dvc init`. It
+contains [`.dvc` directory](/doc/user-guide/project-structure/internal-files),
```

Comment:
> contains `.dvc` directory

How about something like "includes the configuration (see `.dvc/`) and |cache|, as well as your current |workspace| ..."